gotestcover.go

```diff
@@ -235,7 +240,11 @@ func writeCoverProfile(cov []byte) error {
 }
 
 func runGoCommand(args ...string) ([]byte, error) {
-	cmd := exec.Command("go", args...)
+	goCmd := "go"
+	if flagGoAppEngine {
+		goCmd = "goapp"
```

Comment:
ok

The last part: `Flags not recognized by 'go test' must be placed after any specified packages.` is really interesting.
It could be useful to solve #6 .
But I can't find the source code of the `goapp` command.